gax/src/main/java/com/google/api/gax/rpc/BidiStreamingCallable.java

```diff
@@ -44,22 +44,85 @@
 
   protected BidiStreamingCallable() {}
 
+  public abstract ClientStream<RequestT> call(
+      ResponseObserver<ResponseT> responseObserver,
+      ClientStreamCallBack<RequestT> onReady,
+      ApiCallContext context);
+
+  public void call(
```

Comment:
We need versions of each of these methods that don't require an `ApiCallContext`.